pysatNASA/tests/test_omni_hro.py

```diff
@@ -11,8 +11,8 @@ class TestOMNICustom():
     def setup(self):
         """Runs before every method to create a clean testing setup."""
         # Load a test instrument
-        self.testInst = pysat.Instrument('pysat', 'testing', sat_id='12',
-                                         tag='1min', clean_level='clean')
+        self.testInst = pysat.Instrument('pysat', 'testing', tag='',
+                                         num_samples=12, clean_level='clean')
```

Comment:
```suggestion
        self.testInst = pysat.Instrument('pysat', 'testing', tag='',
                                         num_samples=12, clean_level='clean')
        self.testInst.tag = '1min'
```

Reply:
A comment here to remind us why `tag` is being assigned outside instantiation could be helpful for us/others in the future.

Reply:
I would prefer we deal with this in the next version.  The OMNI routines need to be revisited on a larger scale, including testing and organization.  See #34

Reply:
Documented the issue further since this should be made consistant.